livekit_egress.proto

```diff
@@ -60,16 +60,20 @@ message TrackCompositeEgressRequest {
 message TrackEgressRequest {
   string room_name = 1;  // required
   string track_id = 2;   // required
+  bool containerize = 3; // does not work with websocket output
```

Comment:
what does this do?

Reply:
I was confused, if you want containerization you'd use track composite